### ISSUE TITLE
"Suppress" Parameter Mispelled as "Supress" on Several Cmdlets

### ISSUE THREAD

**Issue by natescherer** [EvotecIT/PSWriteExcel#6]
Hi, first off I want to say that this is a fantastic and useful module and I appreciate you writing it.

I just wanted to note that the "Suppress" parameter on several cmdlets is misspelled as "Supress". Not the biggest deal in the world, but could possibly lead to some confusion, so I thought I'd call it out.

**Comment by PrzemyslawKlys** [comment]
Thank you. Unfortunately, I am aware. I made mistake about it in PSWriteWord first and continued in PSWriteExcel, PSWriteHTML, and other modules till I realized that it's actually spelled differently 🤦‍♂️Now when I have time I do fix it - by changing the parameter to proper spelling and adding an alias to "supress" to make sure some code doesn't blow up. It's not a simple replace, because one needs to check if the parameter is used within function somewhere and fix all the spelling. If you're up for the task making a PR that fixes this with alias support - it would be great!
